examples/2_Iterative/03_PDHG.ipynb

```diff
@@ -718,8 +732,8 @@
     "plt.figure(figsize=(30,8))\n",
     "plt.rcParams.update({'font.size': 15})\n",
     "plt.rcParams.update({'lines.linewidth': 5})\n",
-    "plt.plot(fbp_recon.subset(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'FBP')\n",
-    "plt.plot(pdhg_l1.solution.subset(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'L1 regularisation')\n",
+    "plt.plot(fbp_recon.get_slice(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'FBP')\n",
```

Comment:
Can we use assign the row number to a variable as it's used many times:
`row_num = int(ig2D.voxel_num_y/2)`

plt.plot(fbp_recon.get_slice(horizontal_y =row_num).as_array(), label = 'FBP')\n